src/org/opensolaris/opengrok/index/Indexer.java

```diff
@@ -726,23 +726,6 @@ public void prepareIndexer(RuntimeEnvironment env,
                     projects.add(p);
                 }
             }
-
-            // The projects should be sorted...
-            Collections.sort(projects, new Comparator<Project>() {
```

Comment:
It's not obvious to me why it is needed. Possibly for the entries to appear in lexicographic order in configuration.xml. If it's deemed to be necessary, we can always bring it back.